src/RestaurantApi/Users/GetUser.js

```diff
@@ -0,0 +1,10 @@
+import { getAllUsers } from "./GetAllUsers";
+
+export const getUser = async(user_id) => {
+  try{
+    const dishesData = await getAllUsers();
+    return dishesData.find(element => element.user_id === user_id);
```

Comment:
copy paste developer?  :D `dishesData`
and the same like in getDish method, if we need to fetch one doc, fetch only one doc